src/main/java/io/strimzi/utils/Utils.java

```diff
@@ -78,4 +80,14 @@ public static long waitFor(String description, long pollIntervalMs, long timeout
             }
         }
     }
+
+    /**
+     * Static factory method to get a type 4 (pseudo randomly generated) UUID.
+     * @return random Uuid
+     */
+    public static Uuid randomUuid() {
+        UUID uuid = UUID.randomUUID();
+
+        return new Uuid(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
+    }
```

Comment:
Does this work? IIRC Kafka had its own method for the Kraft storage ID. (as long as it does, we might not care)

Reply:
This is basically a sub-set implementation of their method. Kafka uses:
```java
/**
     * Static factory to retrieve a type 4 (pseudo randomly generated) UUID.
     */
    public static Uuid randomUuid() {
        java.util.UUID uuid = java.util.UUID.randomUUID();
        while (uuid.equals(METADATA_TOPIC_ID_INTERNAL) || uuid.equals(ZERO_ID_INTERNAL)) {
            uuid = java.util.UUID.randomUUID();
        }
        return new Uuid(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
    }`
```
and I think we do not need this part for topics:
```java
while (uuid.equals(METADATA_TOPIC_ID_INTERNAL) || uuid.equals(ZERO_ID_INTERNAL)) {
            uuid = java.util.UUID.randomUUID();
        }
```

Reply:
@tombentley Does this sound fine?

Reply:
I mean the part about us not needed the topic checks -> not sure if they are important for the cluster UUID.

Reply:
^^ @tombentley

Reply:
`METADATA_TOPIC_ID_INTERNAL` and `ZERO_ID_INTERNAL` are two sentinel UUIDs. I don't think the `while` check is needed even in the Kafka code, since `java.util.UUID.randomUUID` returns a "type 4" UUID, which by definition has non-zero bytes in places where those sentinels have zero bytes.